tests/minimax.wcon

```diff
@@ -0,0 +1,39 @@
+{
+    "tracker-commons":true,
+    "metadata":{
+           "lab":{"location":"CRB, room 5020", "name":"Behavioural Genomics" },
+           "who":"Firstname Lastname",
+           "timestamp":"2012-04-23T18:25:43.511Z",
+           "temperature":{ "experiment":22, "cultivation":20, "units":"C" },
+           "humidity":{ "value":40, "units":"%" },
+           "dish":{ "type":"petri", "size":35, "units":"mm" },
+           "food":"none",
+           "media":"agarose",
+           "sex":"hermaphrodite",
+           "stage":"adult",
+           "age":"18:25:43.511",
+           "strain":"CB4856",
+           "image_orientation":"imaged onto agar",
+           "protocol":"text description of protocol",
+           "software":{
+                "tracker":{ "name":"Software Name", "version":"1.3.0"},
+                "featureID":"@OMG"
+           },
+           "settings":"Any valid JSON entry with hardware and software configuration can go here"
+     },
+    "units":{"t":"s", "x":"mm", "y":"m"},
+    "data":[
+        {"@OMG":{
```

Comment:
Having extra data on every timepoint for every animal, such as a "z", would indeed by nice to have within the "data" object itself.  So it sounds like we'd like to keep special feature type 1 and type 3, and drop type 2.

Note that the spec allows for units for more than just "x", "y", and "t", so 
```
"data":[
    {"@OMG":{
               "feature_names":["speed", "curvature", "width"],
               "feature_units":["mm/s", "1/mm", "mm"]
    }}
]
```
is unnecessary anyway.  We can just add those units, to the top-level "units" object:

```
"units": {
    "x":"mm",
    "y":"mm",
    "t":"s",
    "speed":"mm/s",
    "curvature":"1/mm",
    "width":"mm"
}
```

So, in summary, I agree that dropping the type 2 special feature would be better.  I've altered the English and JSON specification, test cases, and Python code to reflect this.

@Ichoran please feel free to merge this pull request if you are satisfied with the above.  Thanks.